app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -194,6 +197,15 @@ def patch_remove(group, operation)
       path_string = operation["path"]
 
       if path_string == "members"
+        if operation.key?("value")
+          member_error_check(operation["value"])
+
+          member_ids = operation["value"].map{ |member| member["value"] }
+
+          remove_members(group, member_ids)
+          return
+        end
+
         group.public_send(ScimRails.config.scim_group_member_scope).delete_all
         return
       end
```

Comment:
You can avoid nesting this deeply by just adding another branch to the parent if.
That'll avoid getting "too nesty" with the logic, which gets really difficult to follow if you don't

```suggestion
      if path_string == "members" && operation.key?("value")
        member_error_check(operation["value"])

        member_ids = operation["value"].map{ |member| member["value"] }

        remove_members(group, member_ids)
        return
      elsif path_string == "members"
        group.public_send(ScimRails.config.scim_group_member_scope).delete_all
        return
      end
```